pub-matic-htb.js

```diff
@@ -265,6 +266,34 @@ function PubMaticHtb(configs) {
             secure: Browser.getProtocol() === "https:" ? 1 : 0
         }
 
+        /* ------------------------ Get consent information -------------------------
+         * If you want to implement GDPR consent in your adapter, use the function
+         * ComplianceService.gdpr.getConsent() which will return an object.
+         *
+         * Here is what the values in that object mean:
+         *      - applies: the boolean value indicating if the request is subject to
+         *      GDPR regulations
+         *      - consentString: the consent string developed by GDPR Consent Working
+         *      Group under the auspices of IAB Europe
+         *
+         * The return object should look something like this:
+         * {
+         *      applies: true,
+         *      consentString: "BOQ7WlgOQ7WlgABABwAAABJOACgACAAQABA"
+         * }
+         */
+        var isPrivacyEnabled = ComplianceService.gdpr.isPrivacyEnabled();
```

Comment:
Hi, please use `ComplianceService.isPrivacyEnabled()`; Instead. 
Please refer to: 
https://github.com/indexexchange/ix-base-adapter/blob/d59d71823c70007642923c3d1037d759fccd6ff7/ix-base-htb.js#L176